src/list/src/listScript.sml

```diff
@@ -1076,6 +1082,12 @@ val LENGTH_TL = Q.store_thm
   ‘!l. 0 < LENGTH l ==> (LENGTH (TL l) = LENGTH l - 1)’,
   Cases_on ‘l’ THEN SIMP_TAC arith_ss [LENGTH, TL]);
 
+Theorem LENGTH_TL_LESS_EQ:
```

Comment:
Please use `LE` suffix to stand for less than or equal.

Reply:
Done